datahub-web-react/src/graphql/contract.graphql

```diff
@@ -0,0 +1,53 @@
+fragment contractAssertionParameters on AssertionStdParameters {
+    value {
+        value
+        type
+    }
+    minValue {
+        value
+        type
+    }
+    maxValue {
+        value
+        type
+    }
+}
+
+fragment dataContractDetails on DataContract {
+    urn
+    type
+    properties {
+        freshness {
+            assertion {
+                ...assertionDetails
```

Comment:
please replace `assertionDetails` in all these sections with `assertionDetailsWithRunEvents`

Reply:
Then copy the changes here into `assertion.graphql`:
https://github.com/acryldata/datahub-fork/pull/3343/files#diff-67e2c283a73c986680a1cde81dcd75cf8d31cb4eef3b501c43e54079dbaff39c